freqtrade/exchange/exchange.py

```diff
@@ -135,13 +135,18 @@ def __init__(self, config: Dict[str, Any], validate: bool = True) -> None:
         self._trades_pagination = self._ft_has['trades_pagination']
         self._trades_pagination_arg = self._ft_has['trades_pagination_arg']
 
+        # Leverage properties
         self.trading_mode = TradingMode(config.get('trading_mode', 'spot'))
-
         self.margin_mode: Optional[MarginMode] = (
             MarginMode(config.get('margin_mode'))
             if config.get('margin_mode')
             else None
         )
+        self.liquidation_buffer = config.get('liquidation_buffer', 0.05)
+        if self.liquidation_buffer < 0.0:
+            raise OperationalException('Cannot have a negative liquidation_buffer')
+        if self.liquidation_buffer > 0.99:
+            raise OperationalException('Liquidation_buffer must be below 0.99')
```

Comment:
This is a setting that's coming from the configuration - so it should be checked right there (adding this to `constants.py` - as mentioned here.

please allow the configuration schema to do the validation (you can set min-max values in the configuration).